drivers/hmis/app/graphql/mutations/ce/mark_units_available.rb

```diff
@@ -70,21 +65,5 @@ def build_opportunity_for_unit(unit, template, candidate_pool_resolver)
       opportunity.candidate_pool = candidate_pool_resolver.candidate_pool_for_opportunity(opportunity: opportunity)
       opportunity
     end
-
-    def opportunity_ids_to_update(opportunity_scope:) # Find any opportunities that need to be updated
```

Comment:
I think the change-tracking system should handle this now.

* Cron should re-process all pools every 24 hours already so we shouldn't need to check for stale pools here
* The BuildCandidatePoolsJob task should mark any new pools as dirty